frontend/src/metabase/redux/downloads.ts

```diff
@@ -235,11 +235,24 @@ const getDatasetParams = ({
     if (resource === "question" && token) {
       const params = new URLSearchParams(window.location.search);
 
+      const convertSearchParamsToObject = (params: URLSearchParams) => {
+        const object: Record<string, string | string[]> = {};
+        for (const [key, value] of params.entries()) {
+          if (object[key]) {
+            object[key] = [...object[key], value];
+          } else {
+            object[key] = [value];
+          }
+        }
+
+        return object;
+      };
+
       return {
         method: "GET",
         url: Urls.embedCard(token, type),
         params: new URLSearchParams({
-          parameters: JSON.stringify(Object.fromEntries(params)),
+          parameters: JSON.stringify(convertSearchParamsToObject(params)),
```

Comment:
We mistakenly dropped parameter values when a single parameter has multiple values.

This happens because a parameter could be like this `param=1&param=2` and by using `Object.fromEntries`, we'll only get the last parameter value. We need to perform a merge manually.